### ISSUE TITLE
Tracking Repeat callers

### ISSUE THREAD

**Issue by ryandiks** [natedl98/csr-rh#7]
Does CSR have the ability to flag/recognize a repeat caller?

For example, a client calls in and provides their phone number. At some point in the form submission or confirmation process there is a check on the DB to see if that number has been called from before. A flag is inserted into the ticket notifying of the previous calls with a link to a report view showing the previous calls. The agent can then review the tickets opened in the past.

*** Reviewing tickets probably would occur in the Follow Up form, which is after the hotline intake form. ***

**Comment by natedl98** [comment]
Yeah we have a system that can be enabled that rejects any new submissions from previous callers, however, we don't have this specific alert feature. I like this idea
